### DISCUSSION TITLE
WPF/MacOS equivalent of GLib.UnhandledException

### DISCUSSION THREAD

**Discussion by InfinityGhost** [picoe/Eto#2085] [Q&A] [answered]
Is there a WPF/MacOS equivalent of `GLib.UnhandledException`? I'm aware of `AppDomain.CurrentDomain.UnhandledException`, however I'm wondering if there's a more proper way.

**Comment by cwensley** [comment, marked as answer]
With Eto, use `Application.Instance.UnhandledException`.  It should wire up to the appropriate platform-specific unhandled exception handler.
